### PR TITLE
Input validation

### PR DESCRIPTION
**Pull request by bkaryna**

### CODE REVIEW

File: app/src/main/res/layout/activity_create_party.xml [hunk 1]
```diff
@@ -62,24 +65,25 @@
...
 
                     <com.google.android.material.textfield.TextInputLayout
                         android:id="@+id/PartyDescription"
                         style="@style/Widget.MaterialComponents.TextInputLayout.OutlinedBox"
                         android:layout_width="match_parent"
-                        android:layout_height="100dp"
+                        android:layout_height="wrap_content"
                         android:layout_marginTop="2dp"
+                        app:counterEnabled="true"
+                        app:counterMaxLength="250"
```
Comment (ixrk) [inline app/src/main/res/layout/activity_create_party.xml:86]: 250 characters is too little for a description. Increase it to 500 or 1000.

File: app/src/main/res/layout/activity_create_party.xml [hunk 2]
```diff
@@ -181,7 +188,9 @@
                         android:id="@+id/layoutLocation"
                         style="@style/Widget.MaterialComponents.TextInputLayout.OutlinedBox"
                         android:layout_width="match_parent"
-                        android:layout_height="match_parent"
+                        android:layout_height="wrap_content"
+                        app:counterEnabled="true"
+                        app:counterMaxLength="100"
```
Comment (ixrk) [inline app/src/main/res/layout/activity_create_party.xml:192]: Counter disappears after filling the field.

**Review by ixrk — CHANGES_REQUESTED** [review]
The code is OK, but the layout file is not. 

* Some fields have a hard limit on text length and some don't.
* Text length limits are all over the place. Choose one slightly higher limit for the name, one limit, also higher, for the description and one universal for the rest of fields. It doesn't need to be super optimized.
* Some fields have multiline editing enabled and some don't.

**Review by ixrk — CHANGES_REQUESTED** [review]
It's OK, but increase the max length of a description to 1000.

**Comment by bkaryna** [comment]
> It's OK, but increase the max length of a description to 1000.

💩

**Comment by ixrk** [comment]
> 💩

Have you got some kind of PTSD regarding maximum text length after our database course? 💩

**Comment by bkaryna** [comment]
> > 💩
> 
> Have you got some kind of PTSD regarding maximum text length after our database course? 💩

I guess someone has size issues 👀

**Review by ixrk — APPROVED** [review]

**Pull request merged**